#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_mythmusic_endian.ppc.dpatch by Magnus Ihse Bursie <magnus@ihse.net>
##
## All lines beginning with ## DP:' are a description of the patch.
## DP: Fix endianness problem on PPC in MythMusic.

@DPATCH@
diff -urNad mythplugins~/mythflix/mythflix/netflix-ui.xml mythplugins/mythflix/mythflix/netflix-ui.xml
--- mythplugins~/mythflix/mythflix/netflix-ui.xml	2007-12-09 21:07:23.000000000 -0600
+++ mythplugins/mythflix/mythflix/netflix-ui.xml	2007-12-09 21:07:47.000000000 -0600
@@ -91,7 +91,7 @@
       </image>
 
       <image name="background" draworder="0" fleximage="no">
-        <filename>news-info-bg.png</filename>
+        <filename>flix-info-bg.png</filename>
         <position>0,0</position>
       </image>
 
@@ -172,7 +172,7 @@
         </image>
         
         <image name="background" draworder="0" fleximage="no">
-          <filename>news-info-bg.png</filename>
+          <filename>flix-info-bg.png</filename>
           <position>0,0</position>
         </image>
         
